### DISCUSSION TITLE
What does a segment correspond to ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#1174] [General Q&A] [answered]
Hi,

I hit `At least one sample is shorter than the segment size.` while fine-tuning a model. I achieved to overcome this error by decreasing the `spec_segment_size` from default 32 to 2. But now I wonder what a segment represents and what the consequences of setting `spec_segment_size` as low as 2 are (it seems that training goes very slowly).

Can somebody explain what this is ?

Thanks

**Comment by erogol** [comment, marked as answer]
See section 3.3 https://arxiv.org/pdf/2106.06103.pdf

**Reply by Ca-ressemble-a-du-fake** [reply]
Thank you @erogol now I have to know what a "latent space" is 😄!
